src/services/ParticipantProjectSpeciesService.ts

```diff
@@ -107,7 +107,7 @@ const update = async (
     const index = speciesList.findIndex((ps) => ps.id === participantProjectSpeciesId);
 
     if (index !== -1) {
-      speciesList[index] = participantProjectSpecies;
+      // speciesList[index] = participantProjectSpecies;
```

Comment:
Was this for testing? Does this change need to be included?

Reply:
It was for testing, anyways mocks are removed now I think